packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/method_channel/method_channel_google_maps_flutter.dart

```diff
@@ -134,7 +134,7 @@ class MethodChannelGoogleMapsFlutter extends GoogleMapsFlutterPlatform {
       case 'camera#onMove':
         _mapEventStreamController.add(CameraMoveEvent(
           mapId,
-          CameraPosition.fromMap(call.arguments['position']),
+          CameraPosition.fromMap(call.arguments['position'])!,
```

Comment:
You're using `!` a lot here in ways that don't appear to be safe. There are obvious codepaths that can return null from these methods. Extensive use of `!` is generally an indication that the migration is incomplete or incorrect.